src/delegate.jl

```diff
@@ -9,7 +9,7 @@ macro delegate(source, targets)
         funcname = esc(funcnames[i])
         fdefs[i] = quote
                      ($funcname)(a::($typename), args...) =
-                       ($funcname)(a.($fieldname), args...)
```

Comment:
See https://github.com/JuliaGraphics/Colors.jl/pull/247/files , I think this is not necessary (you can remove the `esc(Expr(:quote))` altogether)